blockbase/inc/block-styles.php

```diff
@@ -1,64 +0,0 @@
-<?php
-/**
- * Blockbase Theme: Block Styles
- *
- * @package Blockbase
- * @since 1.2.8
- */
-
-if ( ! function_exists( 'blockbase_register_block_styles' ) ) :
-
-	function blockbase_register_block_styles() {
-
-		if ( function_exists( 'register_block_style' ) ) {
-
-			/**
-			 * Register block styles
-			 */
-
-			//Post meta styles with icons
```

Comment:
Just noting that adding the ability to toggle icons to post meta blocks is tracked in GB already:

https://github.com/WordPress/gutenberg/issues/30348
https://github.com/WordPress/gutenberg/issues/24957
https://github.com/WordPress/gutenberg/issues/24956
https://github.com/WordPress/gutenberg/issues/24952